rhttpc-transport/src/main/scala/rhttpc/transport/MessagePropertiesNaming.scala

```diff
@@ -21,7 +21,7 @@ object MessagePropertiesNaming {
 
   def attemptProperty = transportProperty("attempt")
 
-  def receiveDateProperty = transportProperty("receive-date")
+  def dateOfFirstAttemptProperty = transportProperty("date-of-first-attempt")
```

Comment:
Can u change property name passed in message to less verbose version: `first-attempt-date`?